openassessment/xblock/static/sass/xb/utilities/_variables.scss

```diff
@@ -39,6 +39,7 @@ $gray-d3: shade($gray,60%);
 $gray-d4: shade($gray,80%);
 $gray-d5: shade($gray,90%);
 $gray-d6: shade($gray,95%);
+$gray-border: rgb(124, 143, 156);
```

Comment:
should be just `gray`

Reply:
that is already declared within variables.